hbase-zookeeper/src/main/java/org/apache/hadoop/hbase/zookeeper/ZKUtil.java

```diff
@@ -168,6 +168,28 @@ public static int checkExists(ZKWatcher zkw, String znode) throws KeeperExceptio
     }
   }
 
+  /**
+   * Get the create time if the specified node exists.
+   * @param zkw   zk reference
+   * @param znode path of node to watch
+   * @return create time of the node if it exists, -1 if does not exist
+   * @throws KeeperException if unexpected zookeeper exception
+   */
+  public static long getCreateTimeIfNodeExists(ZKWatcher zkw, String znode) throws KeeperException {
+    try {
+      Stat s = zkw.getRecoverableZooKeeper().exists(znode, false);
+      return s != null ? s.getCtime() : -1;
```

Comment:
OK, -1, then we'd better have a constant in the ReplicationPeerStorage interface so all the implementations follow the same pattern?